config/locales/views/moderations/fr.yml

```diff
@@ -19,9 +19,19 @@ fr:
           icon: Ajuster les étiquettes
           add: Ajouter une étiquette
           add_placeholder: Ajouter une étiquette
-          reason: Raison de l'ajustement de l'étiquette (optionnel)
+          reason: 
+            remove_tag: Raison de la suppression de la balise (optionnel)
```

Comment:
`Raison de supprimer la balise` also works (and matches the `add_tag` translation) - having shorter translations in general helps